src/cson.coffee

```diff
@@ -60,6 +60,7 @@ parseContents = (objectPath, cachePath, contents, callback) ->
   try
     object = parseObject(objectPath, contents)
   catch parseError
+    parseError.message = "#{objectPath}: #{parseError.message}"
```

Comment:
Would that appear in the error message/stack? 
Can you point to an example please?